frontend/src/scenes/heatmaps/HeatmapsBrowser.tsx

```diff
@@ -0,0 +1,135 @@
+import { LemonBanner, LemonButton, LemonInputSelect, LemonSkeleton, SpinnerOverlay } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { AuthorizedUrlList } from 'lib/components/AuthorizedUrlList/AuthorizedUrlList'
+import { appEditorUrl, AuthorizedUrlListType } from 'lib/components/AuthorizedUrlList/authorizedUrlListLogic'
+import { DetectiveHog } from 'lib/components/hedgehogs'
+import { IconOpenInNew } from 'lib/lemon-ui/icons'
+import { useRef } from 'react'
+
+import { heatmapsBrowserLogic } from './heatmapsBrowserLogic'
+
+export function HeatmapsBrowser(): JSX.Element {
+    const iframeRef = useRef<HTMLIFrameElement | null>(null)
+    const logic = heatmapsBrowserLogic({ iframeRef })
+
+    const { browserSearchOptions, browserUrl, loading, isBrowserUrlAuthorized, topUrls, topUrlsLoading } =
+        useValues(logic)
+    const { setBrowserSearch, setBrowserUrl, onIframeLoad } = useActions(logic)
+
+    return (
+        <div className="flex flex-wrap gap-2">
+            {/* <div className="border rounded bg-bg-light flex-1">
+                <p className="p-2">Controls</p>
+            </div> */}
+
+            <div className="flex flex-col overflow-hidden w-full h-[90vh] rounded border">
+                <div className="bg-accent-3000 p-2 border-b flex items-center gap-2">
+                    <span className="flex-1">
+                        <LemonInputSelect
+                            mode="single"
+                            allowCustomValues
+                            placeholder="e.g. https://your-website.com/pricing"
```

Comment:
you could spark joy by picking a random URL for their team from the sessions table?

Reply:
was reviewing it and then realised.. we just need to smash it in and then pick away at it